storage.conf

```diff
@@ -13,6 +13,10 @@ runroot = "/var/run/containers/storage"
 # Primary Read/Write location of container storage
 graphroot = "/var/lib/containers/storage"
 
+# Storage path for rootless users
+#
+# rootless_storage_path = "$HOME/.config/containers/storage"
```

Comment:
Maybe I've over code-reviewed today, but the man page seems to say this should be `$HOME/.local/share/containers/storage`?

Reply:
Are they talking about no need to specify rootless_storage_path = "$HOME/.config/containers/storage" in storage.conf?  I need ask @rhatdan

Reply:
Yes Tom is correct this should be something like

 rootless_storage_path = "/var/tmp/$USER/containers/storage"

Reply:
/var/tmp? not $HOME/.config/containers/storage? also change thoses man pages to /var/tmp/$USER/containers/storage?

Reply:
Let's talk today.  If we are showing the default then it should be
$HOME/.local/share/containers/storage